SU-SOE/stanford_soe_helper
# SOE-2032 Kerri joe injector

# READY FOR REVIEW

# Summary
- Packaged up all of Kerri's injector rules

# Needed By (Date)
- 8/17 (End of sprint)

# Urgency
- No

# Steps to Test

- Pull this branch
- Verify that styles are being applied
- Note: this branch has been pulled to /jse-soe-dev

# Affected Projects or Products
- SoE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-2032


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
